developer/src/kmc-copy/src/copier-messages.ts

```diff
@@ -231,4 +231,16 @@ export class CopierMessages {
     `The output file '${def(o.filename)}' has two different possible source files.`,
   );
 
+  static VERBOSE_CopyingFile = SevVerbose | 0x0021;
+  static Verbose_CopyingFile = (o:{from: string, to: string}) => m(
+    this.VERBOSE_CopyingFile,
+    `Copying file '${def(o.from)}' to '${def(o.to)}'`,
+  );
+
+  static VERBOSE_CopyingFolder = SevVerbose | 0x0022;
+  static Verbose_CopyingFolder = (o:{from: string, to: string}) => m(
```

Comment:
Do you want to report this in `copyFolder()`?

Reply:
No, I looked at doing that but actually we probably don't need this message at all. Folders are automatically created when files are placed in them.

I think I'll go ahead and remove the message.